packages/ui/src/domainModules/taxon/components/TaxonNameManager/item/EditForm.js

```diff
@@ -44,7 +57,15 @@ export class Edit extends PureComponent {
         formSectionNavigationHeader={`${name} (${moduleTranslate({
           textKey: 'name',
         })})`}
-        initialValues={{ id: itemId, name, rank, rubinNumber }}
+        initialValues={{
+          acceptedToTaxon,
```

Comment:
why add acceptedToTaxon and synonymToTaxon here? I think this causes a bug. the reason for deconstructing certain values from nestedItem instead of sending in the whole nested item is that it will cause problems with updating relations that should not be updated